### DISCUSSION TITLE
Is it possible to add multiple items to cart in one go?

### DISCUSSION THREAD

**Discussion by de-raaf-media** [duncanmcclean/simple-commerce#1099] [Help]
I have a product and I also have some related products. It would be nice to have some checkboxes to select the related product so that you can add - for instance - add a wireless mouse and some batteries (related product) in one go. 

Can this be done currently? Or do I have to add each product via a an Ajax call. So - for instance - for the mouse and batteries I would call something like below twice (or more)?

Ideal would be an array of product ids in the params. But I don't think that is currently possible.

```
let params = {
  _token: "{{ csrf_token }}",
  _request: 'Empty',
  '_error_redirect': 'Empty',
  '_redirect': 'Empty',
  product: "your-product-id",
  quantity: 1,
};
 
axios.post("/!/simple-commerce/cart-items", params).then((response) => {
  console.log("Woo hoo! The product has been added to your cart");
});
```

Regards,
Mark

**Comment by duncanmcclean** [comment]
> Can this be done currently? Or do I have to add each product via a an Ajax call. So - for instance - for the mouse and batteries I would call something like below twice (or more)?

Yes, you'd have to do multiple AJAX requests, one for each product.

I'm not sure how many people need to add multiple products at once.

**Reply by de-raaf-media** [reply]
Well, I found an example. Here, you have a mouse. There is an option to add a mouse pad that works particularly well with this mouse :-) You can select this option with a checkbox, and when you add the mouse to your cart, the mouse pad is added as well.

[Logitech MX Master 3S Graphite](https://www.coolblue.nl/en/product/908518/logitech-mx-master-3s-graphite.html)

However, maybe an Ajax call per product is not such a bad idea, so I think I will go with that.
